pkg/cmd/customrun/delete.go

```diff
@@ -17,42 +17,30 @@ package customrun
 import (
 	"context"
 	"fmt"
-	"os"
 
 	"github.com/spf13/cobra"
 	"github.com/tektoncd/cli/pkg/actions"
 	"github.com/tektoncd/cli/pkg/cli"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
-	"go.uber.org/multierr"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-func crExists(args []string, p cli.Params) ([]string, error) {
-	availableCrs := make([]string, 0)
+func crExists(args string, p cli.Params) error {
```

Comment:
can you please rename the function as `customRunExists`

also can you rename the argument from `args` to `name`